perf/perftool.py

```diff
@@ -48,7 +48,7 @@ def setUp(self):
         # enabler for older runners, but should be removed soon
         elif detected_distro.name in ['rhel', 'SuSE', 'fedora',
                                       'centos', 'redhat']:
-            deps.extend(['perf', 'gcc-g++'])
+            deps.extend(['perf', 'gcc-c++'])
```

Comment:
is this patch for all the rhel based distro or only for rhel8.1 onwards please check

Reply:
@PraveenPenguin This works for all RHEL based distro,  checked repository and I have verified on SLES15 SP2 as well.

Reply:
ah sure I confused with the recent comment on the issue #1630